pandas/core/panel.py

```diff
@@ -856,13 +856,7 @@ def xs(self, key, axis=1):
         if axis == 0:
             return self[key]
 
-        self._consolidate_inplace()
-        axis_number = self._get_axis_number(axis)
-        new_data = self._data.xs(key, axis=axis_number, copy=False)
-        result = self._construct_return_type(new_data)
-        copy = new_data.is_mixed_type
-        result._set_is_copy(self, copy=copy)
-        return result
+        raise NotImplementedError("Panel is removed in pandas 0.25.0")
```

Comment:
Is this code hit? I'm confused by the reference to panel.

Reply:
We've removed most of the Panel tests, but not yet all of Panel itself.  So this is just explicitly-disabling a method that _would_ call BlockManager.xs, since we're on the road towards removing the class anyway.